src/components/NgoList/allNgos.js

```diff
@@ -0,0 +1,61 @@
+import image from "./image/image.jpg";
```

Comment:
name can be more clear

Reply:
This is just a placeholder for now. The real data will come from the database(where there is the image)
Now the functionality and ngo list aren't connected.